app.js

```diff
@@ -0,0 +1,79 @@
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const form = document.querySelector('.form');
+const booksDiv = document.querySelector('.add-books');
+
+function addNewBook(book) {
+  const bookUnit = document.createElement('li');
+  bookUnit.innerHTML = `
+        <p class="book-name">${book.title}</p>
+        <p class="the-auhtor">${book.author}</p>
+        <button class='removeBook' id=${Books.id}>Remove</button>
+        `;
+  booksDiv.appendChild(bookUnit);
+}
+
+function removeBookFromPage(target) {
+  if (target.classList.contains('removeBook')) {
+    target.parentElement.remove();
+  }
+}
+
```

Comment:
- [ ] Changes should be persisted in the local storage. Your remove functionality only removes the book from the page, but not the local storage. So when I refresh the page, the supposedly removed books re-appear on the page. Kindly fix that.

Reply:
@julie-ify Yes that's a good review from you, Thanks